code/game/machinery/iv_drip.dm

```diff
@@ -192,7 +201,8 @@
 /obj/machinery/iv_drip/examine()
 	set src in view()
 	..()
-	if (!(usr in view(2)) && usr!=loc) return
+	if(!(usr in view(2)) && usr != loc)
```

Comment:
Change the start of this proc to:

```
/obj/machinery/iv_drip/examine(mob/user)
    ..()
    if(get_dist(user, src) > 2)
        return
```

```view()``` isn't necessary, as you can't examine something with out being able to see it, and the ```set src in x``` is garbage code and ```usr``` is too but we have ```user``` instead.